cloud/terraform/aws_multi_region/modules/multi-region/variables.tf

```diff
@@ -0,0 +1,57 @@
+variable "admin_ip" {
+  default     = ["127.0.0.1/32"]
+  description = "admin IP addresses in CIDR format"
+}
+
+variable "ec2_ssh_key_name" {
+  default = "default"
+}
+
+# https://aws.amazon.com/ec2/instance-types/
+variable "ec2_instance_type" {
+  default = "t3.xlarge"
+}
+
+# Refer to https://wiki.debian.org/Cloud/AmazonEC2Image/Buster
```

Comment:
Have you updated the AMIs to Bullseye? 22.x will be based on Bullseye.

Reply:
I have not updated them. I see that in the cloud/terraform/aws folder on 22.x branch the same images as below are used.

I have tried however to use the AMIs for Bullseye from here https://wiki.debian.org/Cloud/AmazonEC2Image/Bullseye, but something is wrong when deploying the new AMD AMIs. However when using Buster images all works as expected 😕

Reply:
That is correct. It is a community driven feature that we are not conducting tests with.
Since Buster will be out of support mid-year (and hence not getting updates any longer) I'd recommend using the latest AMIs.

Reply:
Please check if all settings point to the `22.x` branch instead of `master`.

Reply:
Hi @t3chn0m4g3. I have updated the AMIs now and have fixed the cloud-init.yml file to run correctly. I have also tested this and now runs as expected for both old aws terraform deployment and the multi region one.

Reply:
@tmariuss Thank you!
@shaderecker Can you run a final review, please?

Reply:
Yes, looks good to me!